assets/js/blocks/product-tag/block.tsx

```diff
@@ -12,8 +12,7 @@ import {
 	ToolbarGroup,
 	withSpokenMessages,
 } from '@wordpress/components';
-import { Component } from '@wordpress/element';
```

Comment:
Please add a comment similar this:
```
// @ts-expect-error: `ServerSideRender ` currently does not have a type definition in WordPress core
import ServerSideRender from '@wordpress/server-side-render';
```

You can read the benefit of [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) here.

Reply:
What do you think about trying in this way?

```tsx
// @ts-expect-error: `ServerSideRender ` currently does not have a type definition in WordPress core
// eslint-disable-next-line @woocommerce/dependency-group
import ServerSideRender from '@wordpress/server-side-render';
/**
 * External dependencies
 */
```